packages/flutter_markdown/pubspec.yaml

```diff
@@ -7,8 +7,8 @@ issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+
 version: 0.6.13
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
```

Comment:
revert these changes. Good to avoid anything unrelated!

Reply:
Yep, I found it too. Thanks!